components/ExercisePreview.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styles from '../scss/exercisePreview.module.scss'
+
+type exerciseProps = {
+  question: string
+  answers: string[]
+}
+
+type Props = {
+  exercise: exerciseProps
+  classes?: string
+}
+
+const ExercisePreview: React.FC<Props> = ({ exercise, classes }) => {
+  const classesList =
+    classes ||
+    'col-sm-8 col-md-7 col-lg-6 col-xl-5 m-auto px-md-3 border-0 rounded'
+
+  return (
+    <div className={`card shadow-sm ${classesList}`}>
+      <div className="card-body pt-3 pb-3 ">
+        <p className="card-text">
+          <p className={`${styles['exercise__p__bold']}`}>Problem</p>
+          <pre className={`${styles['exercise__pre__bg']} rounded px-4 py-4`}>
+            {exercise.question}
+          </pre>
+          <p className={`${styles['exercise__p__bold']}`}>Choose an answer</p>
+          {exercise.answers.map((answer, index) => (
```

Comment:
What if there are 100 answers? Would it still look good?

Reply:
I guess with 100 answers it wouldn't look good at all, i guess i should probably give it a minimum height.

Reply:
would it make sense to call these options instead of an answer? 

`Choose an answer` <- this prompt is fine.

But `exercise.answers.map` <- this could cause naming issues because every exercise has one correct answer, so when the user picks from this list each element is technically an option for the user, not the "answer"

Reply:
> But `exercise.answers.map` <- this could cause naming issues because every exercise has one correct answer, so when the user picks from this list each element is technically an option for the user, not the "answer"

Yeah, sure! I'll use `options`.